packages/main/test/specs/DatePicker.spec.js

```diff
@@ -351,6 +351,16 @@ describe("Date Picker Tests", () => {
 		datepicker.valueHelpIcon.click()
 		browser.keys("F4");
 
+		assert.notOk(datepicker.calendar.getProperty("_monthPicker")._hidden, "Month picker is open");
```

Comment:
I am not sure that all implemented scenarios are covered with tests

Reply:
Two additional tests are now added regarding the keyboard combinations to open specific picker.

There are two interactions that I can't test as they are only visual:
- CTRL + HOME moves focus to the first day of the displayed month
- CTRL + END moves focus to the last day of the displayed month